lib/application_insights.rb

```diff
@@ -5,5 +5,6 @@
 module ApplicationInsights
   module Rack
     autoload :TrackRequest, "application_insights/rack/track_request"
+    autoload :TrackPageView, "application_insights/rack/track_page_view"
```

Comment:
I realize after merge that this wasn't renamed. Can you send separate PR please?

Reply:
oh, sorry. I'll send new PR later.